modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -74,27 +69,25 @@ types:
       givenName: string
       familyName: string
     examples:
-      fullName: !include fragment1.raml
-      otherFullName: !include fragment2.raml
+      fullName: !include fragment1.yaml
+      otherFullName: !include fragment2.yaml
 ----
 
-.fragment1.raml
+.fragment1.yaml
 ----
-#%RAML 1.0 NamedExample
 givenName: ”Chiaki”
 familyName: "Mukai"
 ----
 
-.fragment2.raml
+.fragment2.yaml
 ----
-#%RAML 1.0 NamedExample
 givenName: "Kyung-won"
 familyName: "Park"
 ----
 
-== Naming two examples for a type in a NamedExample fragment
+== Naming two examples that are in a NamedExample fragment
 
-In the following sample files, the `examples` facet does not name the examples.  The NamedExample fragment names the examples `fullName` and `otherFullName`.
+In the file `api.raml`, the `examples` facet does not name the examples.  The NamedExample fragment names, and it names them the examples `fullName` and `otherFullName`.
```

Comment:
Not sure what is going on in the second sentence: missing word? Or did you mean to add that to the end of the first sentence.